README.md

```diff
@@ -39,8 +39,22 @@ Contributions to this repository follow the default [GitHub workflow](https://gu
 To contribute changes, you can follow these steps:
 
  * Adapt the AsciiDoc files in `src/main/asciidoc` or the website menu in  `src/main/template`.
- * Call `mvn clean compile` to build the site and verify the generated website by viewing it locally with a web browser.
+ * Call `mvn clean install` to build the site and verify the generated website by viewing `target/index.html` locally with a web browser.
```

Comment:
With this new call, the build artifacts are automatically copied to the `docs` directory, which is not part of the `.gitignore`. As a result, such changes could (accidentally) be included in the commit if everything in the workspace is added to the staging area (i.e. by using `git add .`, `git add -u`, or ` git commit -a`).

For this reason, I would suggest still keeping `mvn clean compile` in this section or at least mentioning that the build artifacts are copied to `docs` and should be avoided when adding changes to the staging area.